testsuite/classlibrary/TestVolume.sc

```diff
@@ -1,24 +1,49 @@
 TestVolume : UnitTest {
 
-	test_setVolume {
-		var server, volume, level, temp;
+	var server = Server.default;
+	var volume = Server.default.volume;
+	var level = Server.default.volume.volume;
+	var temp;
```

Comment:
seems to me that a temp variable should not be declared at class scope, and probably deserves a better name

Reply:
you're right. I'll fix that right away.